umbraco/Umbraco-CMS
# Member group (identity role) and store

Implement role functionality for members in .NET Core (identity roles and store). Uses default manager.

Based on the new ASP.NET Core identity implementation for the back office, members, and [UmbracoIdentity](https://github.com/Shazwazza/UmbracoIdentity).

This relates to the larger issue for the whole members migration #9471

### Prerequisites

[ x ] I have added steps to test this contribution in the description below


### Description

Continuing from the Create Member PR #9472 

- No database changes
- Implementation of a store and controller updates for the Members identity implementation
- Adjusted the controller to use the new manager- several questions
- Unit tests for new code

### Testing

1. Run unit tests
1. Run integration tests
1. Login to back-office and create a member with a role. Member should be persisted to the database as it is currently in Umbraco 8, with associated role.
1. Login to back-office and create a role (member group). Role should be persisted to the database as it is currently in Umbraco 8
1. Login to back-office and update a role. Role updates should be persisted to the database as they are in Umbraco 8.
1. Login to back-office and delete a role. Role should be deleted from database
